ansible/roles/deerhide_users/templates/gen-key-script.j2

```diff
@@ -0,0 +1,7 @@
+Key-Type: 1
+Key-Length: 2048
+Subkey-Type: 1
+Subkey-Length: 2048
+Name-Real: {{ ansible_hostname }}
+Name-Email: {{ ansible_hostname }}@deerhide.run
```

Comment:
Note: The conventions haven't been defined yet.

Reply:
it's "artificial" email but hostname.machine@deerhide.run can be good.
( for the moment hostname of melanie is not define :D meaning the result is nus@deerhide.run)

Reply:
Deerhide has a few conventions (e.g., hello@network.deerhide.run), but they can wait, so let's unify them later.